documents/description.md

```diff
@@ -2,8 +2,105 @@
 
 ## Motivation
 
+The goal of the *construct* project is to build a language that allows humans
+to articulate geometric constructions to computers in a natural way.
+
+The general problem here is to design a way for humans to communicate geometry
+to computers in a way which is both geometrically precise and easy for humans
+to produce. I believe that a language is the correct solution because:
+
+   - Language is generally a good solution to communication problems
+   - The language used in geometric/mathematical writing is already reasonably
+     easy to use and unambiguous.
+
+I would classify a language that soles this problem as domain specific because
+it will need to be highly tailored to the geometric domain in order to feel
+easy to use. Furthermore, the domain requires none of the complex flow control
+associated with general purpose languages.
+
 ## Language domain
 
+The domain this language is targeted at is that of classical geometry (as set
+forth by Euclid) on a euclidean plane.
+
+This is the geometry governed by constructions of circles, lines, rays, and
+segments (really just circles and rays) from points.
+
+I imagine this language would be useful to people interested in doing classical
+geometry (students and teachers of geometry classes) and might also end up
+being useful to people interested in making geometric drawings.
+
```

Comment:
I imagine it would be useful then, and in other discipline where geometric drawings are needed.